### ISSUE TITLE
Titles on Subpx Accuracy plots

### ISSUE THREAD

**Issue by Resonanz** [soft-matter/trackpy#387]
Most plots can have titles added to them, but it's not so obvious to me how to set the title on the Subpx plots. Is this possible?

**Comment by caspervdw** [comment]
Yes, the function returns a list of axis objects. This is undocumented, I will mark this so that it will be added. You can set a title with for instance:

    ax[0].set_title('x subpx bias')
